src/preloader/logo.svg

```diff
@@ -1,2 +1,24 @@
-
-<svg width="80" height="80" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient x1="50%" y1="100.845%" x2="50%" y2="2.104%" id="a"><stop stop-color="#165EDB" offset="0%"/><stop stop-color="#99B9FF" offset="100%"/></linearGradient></defs><g fill="none" fill-rule="evenodd"><path d="M22.238 60.188c-7.548 4.451-11.341 6.657-11.379 6.617C-2.814 52.11-3.468 29.162 9.996 13.674a39.655 39.655 0 0 1 28-13.624 1.839 1.839 0 0 1 1.93 1.832l.02 8.915a1.951 1.951 0 0 1-1.897 1.956 26.971 26.971 0 0 0-18.481 9.241c-9.81 11.285-8.614 28.385 2.67 38.194z" fill="url(#a)" fill-rule="nonzero"/><path d="M15.45 55.402l4.596-11.376c.403-1 1.54-1.482 2.54-1.079l8.14 3.29c1 .403 1.483 1.54 1.08 2.54l-5.967 14.766c-.242.6-.532 1.165-.864 1.695a8.777 8.777 0 0 1-7.624 4.42 8.76 8.76 0 0 1-6.63-3.024l-.014-.006-.64-.718a39.478 39.478 0 0 1-6.252-9.664 6.346 6.346 0 0 0 4.845 3.377c0-.004 0-.008.002-.013a6.344 6.344 0 0 0 6.755-4.149.256.256 0 0 1 .032-.059z" fill="#6490F1"/><g><path d="M22.238 60.188c-7.548 4.451-11.341 6.657-11.379 6.617C-2.814 52.11-3.468 29.162 9.996 13.674a39.655 39.655 0 0 1 28-13.624 1.839 1.839 0 0 1 1.93 1.832l.02 8.915a1.951 1.951 0 0 1-1.897 1.956 26.971 26.971 0 0 0-18.481 9.241c-9.81 11.285-8.614 28.385 2.67 38.194z" fill="url(#a)" fill-rule="nonzero" transform="rotate(180 40 40)"/><path d="M64.55 24.598l-4.596 11.376a1.951 1.951 0 0 1-2.54 1.079l-8.14-3.29a1.951 1.951 0 0 1-1.08-2.54l5.967-14.766c.242-.6.532-1.165.864-1.695a8.777 8.777 0 0 1 7.624-4.42 8.76 8.76 0 0 1 6.63 3.024l.014.006.64.718a39.478 39.478 0 0 1 6.252 9.664 6.346 6.346 0 0 0-4.845-3.377c0 .004 0 .008-.002.013a6.344 6.344 0 0 0-6.755 4.149.256.256 0 0 1-.032.059z" fill="#6490F1"/></g></g></svg>
+<svg width="80" height="80" xmlns="http://www.w3.org/2000/svg">
```

Comment:
My svg eyes tell me this is the old icon, is that relevant, or should it be the new thing?
<img width="124" alt="image" src="https://user-images.githubusercontent.com/4631227/151428971-71d09f8e-6c2a-4df7-9bce-588bc93d5ab6.png">